app.py

```diff
@@ -29,11 +29,11 @@ def time_periods():
         return flask.jsonify(time_periods)
 
 
-@app.route('/traffic/<int:segment_id>/<time_parameter>')
-def get_traffic(segment_id, time_parameter):
-    traffic_count = traffic_data.get_traffic_count(segment_id, time_parameter)
+@app.route('/traffic/<int:trailways_id>/<time_parameter>')
+def get_traffic(trailways_id, time_parameter):
```

Comment:
Maybe you could call your objects "trails", so this would be a "trail_id", and Stravas are segments so you could still call it "segment_id".  That keeps you from naming things based on the app -- since trailways could have an ID for other things in the future, like for example `park_id` or something.